_posts/2016-11-09-Rust-1.13.md

```diff
@@ -0,0 +1,469 @@
+---
+layout: post
+title: "Announcing Rust 1.13.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.13.0. Rust is
+a systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.13.0][install] from the appropriate page on
+our website, and check out the [detailed release notes for 1.13.0][notes] on
+GitHub. 1448 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: 
+
+### What's in 1.13.0 stable
+
+The two big things in Rust 1.13 stable are compiler performance and a new
+operator, `?`. As usual, there's also a number of library stabilizations and
```

Comment:
Wording a little awk.  Maybe "Rust 1.13 comes with two big improvements: a new `?` operator that makes error handling easier and fixes to compiler performance improvements of XX%"